modules/core/client/app/init.js

```diff
@@ -1,80 +1,43 @@
-'use strict';
+(function (app) {
+  'use strict';
 
-//Start by defining the main module and adding the module dependencies
-angular.module(ApplicationConfiguration.applicationModuleName, ApplicationConfiguration.applicationModuleVendorDependencies);
+  //Start by defining the main module and adding the module dependencies
+  angular
+    .module(app.applicationModuleName, app.applicationModuleVendorDependencies);
 
-// Setting HTML5 Location Mode
-angular.module(ApplicationConfiguration.applicationModuleName).config(['$locationProvider', '$httpProvider',
-  function ($locationProvider, $httpProvider) {
+  // Setting HTML5 Location Mode
+  angular
+    .module(app.applicationModuleName)
+    .config(bootstrapConfig);
+
+  function bootstrapConfig($locationProvider, $httpProvider) {
     $locationProvider.html5Mode(true).hashPrefix('!');
 
     $httpProvider.interceptors.push('authInterceptor');
   }
-]);
-
-angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, Authentication) {
```

Comment:
Moved to https://github.com/meanjs/mean/pull/1202/files#diff-4f015eca6b0b5f96c11d195d1a2058bc